_docs/tasks/security/mutual-tls.md

```diff
@@ -111,11 +113,14 @@ Note that Istio uses [Kubernetes service account](https://kubernetes.io/docs/tas
 as service identity, which offers stronger security than service name 
 (refer [here]({{home}}/docs/concepts/security/mutual-tls.html#identity) for more information). 
 Thus the certificates used in Istio do not have service name, which is the information that curl needs to verify
-server identity. As a result, we use curl option '-k' to prevent the curl client from verifying service identity
-in server's (i.e., productpage) certificate. 
+server identity. As a result, we use curl option '-k' to prevent the curl client from aborting when failing to
```

Comment:
@wattli I rephrased a bit - what do you think ?

Reply:
lgtm, thanks for fixing this up